build/appveyor/before_build.bat

```diff
@@ -3,13 +3,13 @@
 IF "%PLATFORM%" == "x64" (
   :: SET "QTURL=https://utils.musescore.org.s3.amazonaws.com/qt5120_msvc2017_64.7z"
   :: SET "QTDIR=%cd%\qt\msvc2017_64" & :: uncomment to use our Qt
-  SET "QTDIR=C:\Qt\5.12\msvc2017_64" & :: uncomment to use AppVeyor's Qt
+  SET "QTDIR=C:\Qt\5.9\msvc2017_64" & :: uncomment to use AppVeyor's Qt
   SET "TARGET_PROCESSOR_BITS=64"
   SET "TARGET_PROCESSOR_ARCH=x86_64"
 ) ELSE (
   :: SET "QTURL=https://utils.musescore.org.s3.amazonaws.com/qt5120_msvc2017_32.7z"
   :: SET "QTDIR=%cd%\qt\msvc2017" & :: uncomment to use our Qt
-  SET "QTDIR=C:\Qt\5.12\msvc2017" & :: uncomment to use AppVeyor's Qt
+  SET "QTDIR=C:\Qt\5.9\msvc2015" & :: uncomment to use AppVeyor's Qt

```

Comment:
This means no WebEngine, that requires MSVC 2017 (and in Qt 5.9 also 64bit)

Reply:
I downloaded the [32-bit AppVeyor build](https://ci.appveyor.com/project/shoogle/musescore/builds/22918675) and the WebView in the Start Center works fine, as does Save Online. What makes you think it wouldn't work?